examples/ethernet/ethernet/main/Kconfig.projbuild

```diff
@@ -0,0 +1,17 @@
+menu "Example Configuration"
+
+config PHY_TLK110
```

Comment:
For these kind of options, it's better to use a "choice". Something like this: https://github.com/espressif/esp-idf/blob/master/components/freertos/Kconfig#L14 (note the "choice" keyword starts the series of choices, until the "endchoice" line.)